src/containers/Accounts/AccountNFTTable/AccountNFTTable.tsx

```diff
@@ -28,6 +28,7 @@ export const AccountNFTTable = ({ accountId }: AccountNFTTableProps) => {
       getAccountNFTs(rippledSocket, accountId, pageParam).catch(
         (errorResponse) => {
           const errorLocation = `account NFTs ${accountId} at ${pageParam}`
+          // @ts-ignore
```

Comment:
```suggestion
          // @ts-expect-error
```
So that it's easier to fix later, when we do have TS support for `analytics`

Reply:
i actually fixed the issue - must have forgotten to remove the ignore. removing this